Rakefile

```diff
@@ -14,7 +14,8 @@ def bundle_options
 end
 
 def each_database(&block)
-  ['sqlite', 'postgres', 'mysql'].each &block
+  #['sqlite', 'postgres', 'mysql'].each &block
+  ['sqlite'].each &block
```

Comment:
```suggestion
  ['sqlite', 'postgres', 'mysql'].each &block
```

Reply:
I would put the burden of multi-db installation upon the CI, #120 has the multi db config commented out which can easily be enabled. 

If a developer doesnt have mysql or postgres this should be fine. Would should allow them to contribute easily.